spec.js

```diff
@@ -0,0 +1,37 @@
+
+//spec.js
+browser.ignoreSynchronization = true;
```

Comment:
If we test against a non-angular site it to set browser.ignoreSynchronization=true.